### ISSUE TITLE
Escape sequences and printf placeholders get expanded in "Found..." message

### ISSUE THREAD

**Issue by cupnoodles14** [MichaelAquilina/zsh-you-should-use#98]
# Issue Details
## Operating System (uname -a)

`Linux soba 5.4.39-1-MANJARO #1 SMP PREEMPT Wed May 6 10:36:44 UTC 2020 x86_64 GNU/Linux`

## zsh version (zsh --version)
`zsh 5.8 (x86_64-pc-linux-gnu)`

## you-should-use version (echo "$YSU_VERSION")
`1.7.0`

## How is zsh-you-should-use installed?

- [ ] zplug
- [ ] oh-my-zsh
- [ ] Antigen
- [x] Other ([zinit](https://github.com/zdharma/zinit))

## Steps to reproduce the issue

1. Add alias.

```sh
alias pf="printf '%s\n'"
```
2. Run `printf '%s\n' 1 2 3`.

3. Get unexpected output.
```sh
$ printf '%s\n' 1 2 3
Found existing alias for "printf '
'". You should use: "pf"
1
2
3
```

## Minimal reproducible example .zshrc

```sh
source "$HOME/.zinit/bin/zinit.zsh"
zinit wait lucid for MichaelAquilina/zsh-you-should-use
alias pf="printf '%s\n'"
```

**Comment by MichaelAquilina** [comment]
@cupnoodles14 this issue should be fixed in the latest release (1.7.1). Update using your plugin of choice and let me know if its fixed :)

**Comment by cupnoodles14** [comment]
Works great, thanks!
